type.go

```diff
@@ -127,5 +128,15 @@ func fixup(typ *Type, q *Query) {
 	//   GeoAdd(string, []*redis.GeoLocation) *redis.IntCmd
 	//
 	// Should be fixed layer below, in type.go.
+
+	// when include other package struct
+	if typ.ImportPath != "" && typ.ImportPath != q.Package && typ.IsComposite {
+		idx := regexp.MustCompile(`\w`).FindStringIndex(typ.Name)
```

Comment:
Make `regexp.MustCompile(`\w`)` a global var, so it's not compiled each function cal